src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelCreationService.java

```diff
@@ -1,25 +1,25 @@
 package uk.gov.hmcts.ethos.replacement.docmosis.service.excel;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.util.List;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.poi.ss.usermodel.*;
 import org.apache.poi.ss.util.CellRangeAddressList;
 import org.apache.poi.xssf.usermodel.XSSFRow;
 import org.apache.poi.xssf.usermodel.XSSFSheet;
 import org.apache.poi.xssf.usermodel.XSSFWorkbook;
 import org.springframework.stereotype.Service;
-import static uk.gov.hmcts.ecm.common.model.multiples.MultipleConstants.*;
 import uk.gov.hmcts.ecm.common.model.multiples.MultipleObject;
 import uk.gov.hmcts.ethos.replacement.docmosis.helpers.MultiplesHelper;
 
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.ecm.common.model.multiples.MultipleConstants.*;
```

Comment:
CheckStyle again

Reply:
I know checkstyle would normally complain about this but this is what it was already set as. Angela has simply moved it down a couple of lines. Been like that for 3 months!